### PR TITLE
Upgrade Flink version to 1.9 + readme #4

### PR DESCRIPTION
**Pull request by aleksanchezz**
Minimal changes in code and pom.
Code was tested on our test stand, comparing to master branch --- no data loss neither performance degradation was detected

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,11 +27,11 @@
...
-        <flink.version>1.3.2</flink.version>
+        <scala.version>2.11</scala.version>
+        <flink.version>1.9.0</flink.version>
 
         <!-- Secondary components -->
-        <async.client.version>2.0.39</async.client.version>
+        <async.client.version>2.10.4</async.client.version>
         <logback.version>1.2.3</logback.version>
         <guava.version>23.0</guava.version>
         <typesafe.config.version>1.3.3</typesafe.config.version>
```
Comment (mchernyakov) [inline pom.xml]: Probably we should remove this dependency or put it in test scope, cause we use it only in tests.

File: pom.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
     <groupId>ru.ivi.opensource</groupId>
     <artifactId>flink-clickhouse-sink</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.1.0</version>
```
Comment (mchernyakov) [inline pom.xml]: I suggest to set it 1.1.0-SNAPSHOT (if I am not mistaken, deploy-plugin requires it)

File: README.md [hunk 3]
```diff
@@ -141,4 +142,4 @@ dataStream.map(YourEventConverter::toClickHouseInsertFormat)
 ## Roadmap
 - [x] maven
 - [ ] reading files from "failed-records-path"
-- [ ] Flink 1.7+
\ No newline at end of file
+- [x] Flink 1.7+ (Flink 1.9.0)
```
Comment (mchernyakov) [inline README.md]: probably we can remove it :)

File: pom.xml (hunk 2 above)
Comment (aleksanchezz) [reply pom.xml]: done

**Pull request merged**